README.md

```diff
@@ -126,7 +128,7 @@ To check if your fetch request is working, you can look at the `db.json` file to
 
 **YOUR NOTES**
 ```
-
+I originally had newConst outside of the event listener, but it kept returning NaN and causing the db to persist a null value for the likes. Once I moved it inside, it worked correctly. I couldn't figure out why :(
```

Comment:
This makes sense if you think about the timing of when that variable gets set. If you have the variable outside the event listener, it will get set to whatever the value is when the page first loads: `parseInt("{player likes} Likes")`. If you move it in the event listener, it will only set the value when the button is clicked, at which point in time the text will have the current number of likes: `parseInt("1000 Likes")`